Packs/SophosCentral/Integrations/SophosCentral/SophosCentral_test.py

```diff
@@ -1,4330 +1,4331 @@
-import json
-from datetime import datetime
-import pytest
-import requests
-from pytest import raises
-import demistomock as demisto
-from CommonServerPython import (
-    DemistoException,
-    EntryType,
-    set_integration_context,
-    get_integration_context,
-)
-from SophosCentral import Client

```

Comment:
Please revert the deletion and add only the relevant changes

Reply:
Hi sapirshuker, I don't quite understand the nature of this error. Would you be so kind as to explain the error to me better? Thank you